tools/src/main/java/org/apache/kafka/tools/KafkaClientPerformance.java

```diff
@@ -274,28 +291,54 @@ private static ArgumentParser argParser() {
                 .help("print out metrics at the end of the test.");
 
         parser.addArgument("--transactional-id")
-               .action(store())
-               .required(false)
-               .type(String.class)
-               .metavar("TRANSACTIONAL-ID")
-               .dest("transactionalId")
-               .setDefault("performance-producer-default-transactional-id")
-               .help("The transactionalId to use if transaction-duration-ms is > 0. Useful when testing the performance of concurrent transactions.");
+                .action(store())
```

Comment:
I think we need to remove the leading space here and on the next three sections? It would be good to not change the line of code if the only difference is whitespace. It will also keep your name out of `git blame` unnecessarily :).

Reply:
Lol you remind me of the Linkedin style.